arch/cortex-m0/src/lib.rs

```diff
@@ -77,7 +73,7 @@ pub unsafe extern "C" fn switch_to_user(mut user_stack: *const u8,
 
     "
     : "=r"(user_stack)
-    : "r"(user_stack), "r"(process_got), "r"(process_regs)
+    : "r"(user_stack), "r"(process_regs)
```

Comment:
I don't think this would actually effect emitted code, but I think `r0` should be listed as a clobber as well on the line below